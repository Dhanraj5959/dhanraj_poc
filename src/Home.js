import React from "react";

const Home = () => {
  return (
    <>
      <h1 className="text-light" style={{ margin: "200px" }}>
        <b>Welcome to D-Bank</b>
        <br />
      </h1>
    </>
  );
};

export default Home;
