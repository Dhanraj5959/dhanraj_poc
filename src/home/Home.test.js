import { render, screen } from "@testing-library/react";

import Home from "./Home";
import { renderComponent } from "../utils/test_helpers";

describe("Home page", () => {
  test("Should Render Intial Page", () => {
    render(renderComponent(<Home />));
    const text1 = screen.getAllByText(/Welcome to D-Bank/i);
  });
  test("Matching the snapshot of the Home", async () => {
    const { asFragment } = render(renderComponent(<Home />));
    expect(asFragment()).toMatchSnapshot();
  });
});
