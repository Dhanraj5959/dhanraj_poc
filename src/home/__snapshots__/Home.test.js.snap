// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page Matching the snapshot of the navbar 1`] = `
<DocumentFragment>
  <h1
    class="text-light"
    style="margin: 200px;"
  >
    <b>
      Welcome to D-Bank
    </b>
    <br />
  </h1>
</DocumentFragment>
`;
